src/coreclr/nativeaot/Runtime/rhassert.h

```diff
@@ -60,8 +60,12 @@ void Assert(const char * expr, const char * file, unsigned int line_num, const c
     ASSERT_UNCONDITIONALLY(message); \
     ASSUME(0);  \
 
-#define FAIL_FAST_GENERATE_EXCEPTION_ADDRESS 0x1
+//#define FAIL_FAST_GENERATE_EXCEPTION_ADDRESS 0x1
```

Comment:
Why is this commented out?

Reply:
That should be removed. I was seeing if I could remove it. Thanks!